pixl_core/src/core/omop.py

```diff
@@ -17,24 +17,38 @@
 import logging
 import pathlib
 import shutil
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, Any
 
+import pandas as pd
 import slugify
 
 if TYPE_CHECKING:
     import datetime
 
+
 root_from_install = pathlib.Path(__file__).parents[3]
 
 logger = logging.getLogger(__file__)
 
 
-class OmopExtract:
-    """Processing Omop extracts on the filesystem."""
+class ParquetExport:
+    """Exporting Omop and Emap extracts to Parquet files."""
+
+    root_dir: pathlib.Path = root_from_install
 
-    def __init__(self, root_dir: pathlib.Path = root_from_install) -> None:
-        """Create instance of OMOPExtract helper."""
-        self.export_dir = root_dir / "exports"
+    def __init__(self, project_name: str, extract_datetime: datetime.datetime) -> None:
+        """
+        :param project_name: name of the project
+        :param extract_datetime: datetime that the OMOP ES extract was run
+        :param root_dir: export directory root
+        """
```

Comment:
I like this change a lot. Much more clean to do set this up as part of the intialisation

Reply:
Although I've just noticed that `root_dir` no longer exists - our linting should be configured to catch this.